docs/manual/cert.md

```diff
@@ -1,13 +1,52 @@
-As more and more services move to "secure by default" behavior, it becomes important to simplify the PKI/TLS infrastructure provisioning in the lab environments. Containerlab embeds parts of [cfssl](https://github.com/cloudflare/cfssl) project to automate certificate generation and provisioning.
+As more and more services move to "secure by default" behavior, it becomes important to simplify the PKI/TLS infrastructure provisioning in the lab environments.  
+Containerlab tries to ease the process of certificate provisioning providing the following features:
 
-For [SR Linux](kinds/srl.md) nodes containerlab creates Certificate Authority (CA) and generates signed cert and key for each node of a lab. This makes SR Linux node to boot up with TLS profiles correctly configured and enable operation of a secured management protocol - gNMI.
+1. Automated certificate provisioning for lab nodes.
+2. Simplified CLI for CA and end-node keys generation.
+3. Ability to use custom/external CA.
+
+## Automated certificate provisioning
+
+Automated certificate provisioning is a two-stage process. First, containerlab creates a Certificate Authority (CA) and generates a certificate and key for it, storing these artifacts in a [lab directory](conf-artifacts.md) in the `.tls` directory. Then, containerlab generates a certificate and key for each node of a lab and signs it with the CA. The signed certificate and key are then installed on the node.
 
 !!!note
-    For other nodes the automated TLS pipeline is not provided yet and can be addressed by contributors.
+    Currently, automated installation of a node certificate is implemented only for [Nokia SR Linux](kinds/srl.md).
+
+### CA certificate
+
+When generating CA certificate and key, containerlab can take in the following optional parameters:
+
+* `.settings.certificate-authority.key-size` - the size of the key in bytes, default is 2048
+* `.settings.certificate-authority.validity-duration` - the duration of the certificate. For example: `10m`, `1000h`. Max unit is hour. Default is `8760h` (1 year)
+
+### Node certificates
+
+The decision to generate node certificates is driven by either of the following two parameters:
 
-Apart from automated pipeline for certificate provisioning, containerlab exposes the following commands that can create a CA and node's cert/key:
+1. node kind
+2. `issue` boolean parameter under `node-name.certificate` section.
+
+For SR Linux nodes the `issue` parameter is set to `true` by default. For other node kinds the `issue` parameter is set to `false` by default and can be [overridden](nodes.md#certificate) by the user.
```

Comment:
Not just by default, but forcefully set to true.

Reply:
agree, fixed